packages/server/src/fhir/operations/export.ts

```diff
@@ -44,21 +44,28 @@ export async function bulkExportHandler(req: Request, res: Response): Promise<vo
   res.set('Content-Location', `${baseUrl}fhir/R4/bulkdata/export/${bulkDataExport.id}`).status(202).json(accepted);
 }
 
-async function exportResourceType(exporter: BulkExporter, project: Project, resourceType: ResourceType): Promise<void> {
+export async function exportResourceType(
+  exporter: BulkExporter,
+  resourceType: ResourceType,
+  maxResources = 1000
+): Promise<void> {
   const repo = exporter.repo;
   const hasMore = true;
```

Comment:
I just noticed that `hasMore` is a `const` :(

Sonar will not allow `while (true)`.

The better fix is to use `let hasMore = true;`, and then down below, `hasMore = !!linkNext`.